docs/SUMMARY.md

```diff
@@ -42,3 +42,4 @@
 * [CNM Workflow](data-cookbooks/cnm-workflow.md)
 * [SIPS Workflow](data-cookbooks/sips-workflow.md)
 * [Error Handling](data-cookbooks/error-handling.md)
+* [Choice States](data-cookbooks/choice-states.md)
```

Comment:
I think this change should also be added to the `README.md` as well, at least until https://github.com/nasa/cumulus/pull/513 gets merged. Otherwise there will be no link on the doc homepage.

Reply:
Oh yeah I'm trying to merge mine and then you will be all good 👍